src/dmd/target.d

```diff
@@ -87,6 +88,14 @@ Target.OS defaultTargetOS()
     else
         static assert(0, "unknown TARGET");
 }
+uint defaultFreeBSDVersion()
+{
+    version (FreeBSD_12)       return 12;
+    else version (FreeBSD_11)  return 11;
+    else version (FreeBSD_10)  return 10;
+    else version (FreeBSD)     return 11;
+    else                       return 0;
```

Comment:
Maybe the last `else` should result in a `static assert(0)`

Reply:
No. We want to be able to compile on systems other than `FreeBSD` :)